<launch>
    <!-- Khởi tạo tham số -->
    <rosparam param="imgsz">"[640, 640]"</rosparam>
    
    <rosparam param="value_reset_link_0">1</rosparam>
    <rosparam param="angle_reset_link_1">90</rosparam>
    <rosparam param="angle_reset_link_2">0</rosparam>

    <!-- Khởi chạy node yolov5_sort.py -->
    <node
        name="yolov5_sort_node"
        pkg="yolov5_sort"
        type="detect.py"
        output="screen">
        <param name="weights" value="$(env TRACKING_DRONE_WS)/launch/yolov5n.pt"/>
        <param name="data" value="$(env TRACKING_DRONE_WS)/launch/coco128.yaml"/>
        <param name="flip_image" value="0"/>
    </node>

    <!-- Khởi chạy rosbridge_websocket -->
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
        <arg name="port" value="9090"/>
    </include>

</launch>
